Modules/Core/src/Controllers/Gateways/StripeController.php

```
@@ -64,7 +64,7 @@ public function createCheckoutSession($invoice_url_key)
         $invoice = $this->invoicesService->where('invoice_url_key', $invoice_url_key)->first();
         // Check if the invoice is payable
         if ($invoice->invoice_balance <= 0) {
-            $this->session->set_userdata('alert_error', lang('invoice_already_paid'));
+            session(['alert_error', lang('invoice_already_paid'));
```

PHPStan (2.1.31)

Syntax error, unexpected ')', expecting ']' on line 67


(phpstan.parse)